app/assets/json/translations/locale-en.json

```diff
@@ -627,6 +627,42 @@
         "DUE_TO_HIGH_CALL_VOLUME": "Due to high call volume you may hear a message.",
         "OFFICE_HOURS": "Office Hours"
     },
+    "LISTINGS_FOR_RENT": {
```

Comment:
@chrisdolendo I tried running `grunt translations` to keep everything nice and alphabetical, but I ended up with a pretty large diff beyond what's in this PR, so I kept it as-is.

Would it make sense to run `grunt-translations` on a branch off master in a separate story?